### DISCUSSION TITLE
Is it planned to support WebStorm/PHPStorm IDEs?

### DISCUSSION THREAD

**Discussion by edimitchel** [nuxtrdev/nuxtr-vscode#114] [Questions]
In my company, we chose PHP Storm to work on PHP stuff.
We did a decision for futures applications to use Nuxt, but the support is not quite smooth as we could have on VSCode.

I'm questioning me if nuxtr will be supporting JetBrains IDEs as Volar is doing it for both environment.

**Comment by adhamfarrag** [comment]
Hi @edimitchel :) 

Generally I'm considering other IDE support, but we're between having un finished roadmap for VSCode, and the question of consistency between what extension/plugin APIs offer across IDEs and offering a solid DX across all supported IDEs. 

Personally, I can't wait for [Zed](https://zed.dev) to open source their plugin APIs (spamming @JosephTLyons out of love for the project) and they somehow said that I may not be able to offer the same features by API design. 

Hopefully, you'll be able to use Nuxtr on WebStorm or Zed by the end of third quarter this year.
